src/components/HotelMoreDetails.jsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react'
+import { useEffect, useState } from 'react'
+import { fetchData } from '../utils'
+import { makeStyles } from '@material-ui/core/styles'
+import Grid from '@material-ui/core/Grid'
+import Container from '@material-ui/core/Container'
+import LoadingIcon from '../content/LoadingIcon'
+import { RoomCard } from './RoomCard'
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    width: '100%',
+    height: '200px',
+    display: 'flex',
+    flexWrap: 'wrap',
+    justifyContent: 'center',
+    backgroundColor: 'rgb(236 236 236)',
+    margin: '5px',
+    borderRadius: '5px',
+    boxSizing: 'border-box',
+    padding: '10px',
+    '& > span': {
+      margin: theme.spacing(2),
+    },
+  },
+  header: {
+    fontSize: '22px',
+  },
+}))
+
+export function HotelMoreDetails(props) {
+  const classes = useStyles()
+  const [hotel, setHotel] = useState(null)
+  const [loading, setLoading] = useState(true)
+
+  const hotelId = props.match.params.id
```

Comment:
lol just pass hotelId and city as props - { hotelId, city } , dont need to do that like you did